src/components/campaign-wizard/FormHeading.jsx

```diff
@@ -9,7 +9,12 @@ import Icon from "@material-ui/core/Icon";
 function FormHeading({ variant, title, onBackClick }) {
     return (
         <div>
-            <IconButton onClick={onBackClick} color="secondary" aria-label={i18n.t("Back")}>
+            <IconButton
+                onClick={onBackClick}
+                color="secondary"
+                aria-label={i18n.t("Back")}
+                style={{ marginBottom: 7 }}
```

Comment:
This style still being used inline. @matiasgarcia91 can you please extract it to a class?

Reply:
Is a constant style object enough? This is a functional component not wrapped with the material-ui `withStyles`

Reply:
Yes, we tend to use withStyles for new components, but using a styles object is also fine.